packages/message-box-react/src/MessageBox.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { ReactNode } from "react";
+import { H4, P } from "@fremtind/jkl-typography-react";
+
+interface Props {
+    title: string;
+    children: ReactNode;
+}
+
+type messageTypes = "info" | "error" | "success";
+
+function messageFactory(messageType: messageTypes) {
+    return function messageBox({ title, children }: Props) {
+        return (
+            <div className={`jkl-message-box__${messageType}`}>
```

Comment:
Not BEM, should also have a "plain" `jkl-message-box`

Reply:
Bemified it